client/src/components/DisplayTable/DisplayTable.js

```diff
@@ -266,6 +282,31 @@ export class DisplayTable extends React.Component {
       .reverse()
       .value();
 
+    const paginated_data = _.chunk(
+      sorted_filtered_data,
+      disable_pagination ? _.size(sorted_filtered_data) : page_size
+    );
+
+    const number_of_pages = paginated_data.length;
+
+    const change_page_size = (new_page_size) => {
+      this.setState({ page_size: new_page_size, current_page: 0 });
+    };
+
+    const change_page = (page) => {
+      this.setState({ current_page: page });
+      this.first_data_ref.current.focus();
+    };
```

Comment:
General React best practice, functions used as DOM event callbacks shouldn't be declared inside the render, should be class methods.

In JS, functions can't be compared by content, only reference, e.g.
```
const a = () => {};
const b = () => {};
const c = a;

a === b; // false
a === c; // true
```

So, if DOM event callbacks are re-declared every render, React has to assume they've changed and will need to touch the DOM to update them on every render. Not huge, but something worth keeping in mind!